### PR TITLE
Fix map rotation UI for skirmish maps

### PR DESCRIPTION
**Pull request by cemathey**
This is super quick and dirty and gross (but it does work) and we should come back to it after the API returns more useful map information (thanks Abu)

image

### CODE REVIEW

**Review by ElGuillermo — APPROVED** [review]

File: rcongui/src/components/MapManager/DraggableListItem.js [hunk 1]
```diff
@@ -31,7 +31,10 @@ const DraggableListItem = ({ item, index, onRemove }) => {
 
     if (fullName.search("offensive") !== -1 || fullName.search("off") !== -1) {
       labels.push("offensive");
-    } else {
+    } else if (fullName.toLowerCase().search('skirmish') !== -1) {
```
Comment (FlorianSW) [inline rcongui/src/components/MapManager/DraggableListItem.js:34]: there is `String.prototype.includes` for that :D

File: rcongui/src/components/MapManager/DraggableListItem.js [hunk 2]
```diff
@@ -43,7 +46,7 @@ const DraggableListItem = ({ item, index, onRemove }) => {
     if (fullName.toLowerCase().endsWith("ger")) {
       labels.push("axis");
     }
-    if (fullName.search("night") !== -1) {
+    if (fullName.toLowerCase().search("night") !== -1) {
```
Comment (FlorianSW) [inline rcongui/src/components/MapManager/DraggableListItem.js:49]: ah, copy&paste :D We should probably use includes here as well.

File: rcongui/src/components/Scoreboard/utils.js [hunk 3]
```diff
@@ -1,12 +1,25 @@
 export const UNKNOWN_MAP_NAME = 'unknown';
 
+export const SKIRMISH_MAPS = [
```
Comment (FlorianSW) [inline rcongui/src/components/Scoreboard/utils.js:3]: do we need this array? 🤔

Comment (cemathey) [reply rcongui/src/components/Scoreboard/utils.js]: No, I think I added that when I was originally messing around with this and forgot all about it lol.

File: rcongui/src/components/MapManager/DraggableListItem.js (hunk 2 above)
Comment (cemathey) [reply rcongui/src/components/MapManager/DraggableListItem.js]: Much better, thanks! I am so JS ignorant.

**Comment by cemathey** [comment]
Updated built and tested, still works fine.

Thanks! Sorry I neglected to read the github emails and didn't realize there were comments.

**Pull request merged**